src/plugins/plot/inspector/PlotOptionsBrowse.vue

```diff
@@ -26,7 +26,7 @@
 >
     <ul
         v-if="!isStackedPlotObject"
-        class="c-tree"
+        class="c-tree js-series-properties"
```

Comment:
we should consider using an `aria-label` here instead of an extra class if used solely for e2e

Reply:
I plan to use this for adding unit tests at a later point.

Reply:
We should add it then when you add the unit tests instead of adding an unused class, but also we should update the e2e tests to not be reliant on this.

if it has an accessible name (label, aria-label, etc) then we can use role selectors to deterministically select on this without the need of another class:

```js
expect(page.getByRole('list', { name: "Plot series properties" }).locator('h2')).toContainText("Plot Series");
```